apps/emqx_authz/test/emqx_authz_redis_SUITE.erl

```diff
@@ -4,7 +4,8 @@
 %% Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This file seems very similar to that in #6446 , with few differences.
Maybe it needs merging/rebasing?